src/api/ConfigurationService.ts

```diff
@@ -0,0 +1,7 @@
+import {GetRequest, GetResponse } from '.';
+
+export interface ConfigurationService<TGetReq, TGetResp> {
```

Comment:
I think we need list method as well
Show all the entries available for the user/token